apps/dialtone-documentation/docs/components/split-button.md

```diff
@@ -1,1480 +1,636 @@
-                  <dt-button size="lg" importance="outlined" kind="muted" class="d-split-btn__alpha d-split-btn__alpha--lg">
-                    <template #icon>
-                      <dt-icon name="phone" size="400" />
-                    </template>
-                  </dt-button>
-                  <dt-button size="lg" importance="outlined" kind="muted" class="d-split-btn__omega d-split-btn__omega--lg" aria-label="More options">
-                    <template #icon>
-                      <dt-icon name="chevron-down" size="200" />
-                    </template>
-                  </dt-button>
-                </span>
-              </div>
-              <div class="d-ta-center">
-                <span class="d-split-btn">
-                  <dt-button size="xl" importance="outlined" kind="muted" class="d-split-btn__alpha d-split-btn__alpha--xl">
-                    <template #icon>
-                      <dt-icon name="phone" size="500" />
-                    </template>
-                  </dt-button>
-                  <dt-button size="xl" importance="outlined" kind="muted" class="d-split-btn__omega d-split-btn__omega--xl" aria-label="More options">
-                    <template #icon>
-                      <dt-icon name="chevron-down" size="300" />
-                    </template>
-                  </dt-button>
-                </span>
-              </div>
-            </dt-stack>
-          </dt-stack>
-        </td>
-        <td class="d-ta-center">
-          <abbr class="d-fc-muted d-td-none d-fs-100" title="Not applicable">N/A</abbr>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-  <div class="d-p16">
-    <dt-stack direction="row" gap="600" class="d-jc-center">
-      <dt-toggle size="sm" onclick="const splitButtons = document.querySelectorAll('.d-split-btn'); splitButtons.forEach(splitButton => { const buttons = splitButton.querySelectorAll('.d-btn'); buttons.forEach(button => { if (button.hasAttribute('disabled')) { button.removeAttribute('disabled'); } else { button.setAttribute('disabled', 'true'); } }); });">
-        <div class="d-mr6">
-          Disable all split buttons
-        </div>
-      </dt-toggle>
-      <dt-button importance="outlined" icon-position="left" onclick="var element=document.querySelector('.asdfasdfqwerqwer');if(element){element.closest('.asdfasdfqwerqwer').setAttribute('hidden','');}">
-        Hide grid
-        <template #icon>
-          <dt-icon
-            name="eye"
-            size="300"
-          />
-        </template>
-      </dt-button>
-    </dt-stack>
+<code-well-header>
+  <dt-split-button>
+    Place call
+  </dt-split-button>
+</code-well-header>
+
+## Usage
+
+## Variants
+
+### Base
+
+<code-well-header>
+  <div class="d-d-flex d-flow8">
+      <dt-split-button> Place Call </dt-split-button>
+      <dt-split-button importance="outlined"> Place Call </dt-split-button>
+      <dt-split-button importance="clear"> Place Call </dt-split-button>
```

Comment:
Add a Tooltip for all icon-only omega buttons on this page, especially since they're required. e.g. "More calling options"